api/tests/hours_test.py

```diff
@@ -0,0 +1,30 @@
+from django.test import TestCase
+from api.models import Hour, Location
+
+
+class HourTest(TestCase):
+    sample_location_data = {'name': 'Youth Green',
+                            'category': 'Community Garden',
+                            'address_1': '12467 W. Osborne Street',
+                            'city': 'Los Angeles',
+                            'state': 'CA',
+                            'zipcode': '91331',
+                            'latitude': 34.25777331,
+                            'longitude': -118.4046685}
+    sample_hour_data = {'day': 'AllMo',
+                        'open_time': '08:00',
+                        'close_time': '14:00',
+                        }
+
+    def test_creating_hour(self):
+        loc = Location(**self.sample_location_data)
+        loc.save()
+        self.sample_hour_data['location_id'] = loc.id
+        hour = Hour(**self.sample_hour_data)
```

Comment:
That's a good point. I totally see where you are coming from. I'll go ahead and re-work and re-submit